wls-basisdaten-service/src/test/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/domain/WahltagRepositoryTest.java

```diff
@@ -0,0 +1,88 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.domain;
+
+import static de.muenchen.oss.wahllokalsystem.basisdatenservice.TestConstants.SPRING_NO_SECURITY_PROFILE;
+import static de.muenchen.oss.wahllokalsystem.basisdatenservice.TestConstants.SPRING_TEST_PROFILE;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Optional;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.MicroServiceApplication;
+import java.util.Comparator;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest(
+        classes = { MicroServiceApplication.class },
+        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+@ActiveProfiles(profiles = { SPRING_TEST_PROFILE, SPRING_NO_SECURITY_PROFILE })
+@Slf4j
+class WahltagRepositoryTest {
+
+    @Autowired
+    private WahltagRepository repository;
+
+    @AfterEach
+    void tearDown() {
+        repository.deleteAll();
+    }
+
+    @Test
+    void findAll() {
+        List<Wahltag> wahltageToSave1 = createWahltagList("1");
+        List<Wahltag> wahltageToSave2 = createWahltagList("2");
+        List<Wahltag> wahltageToSave3 = createWahltagList("3");
+        repository.saveAll(wahltageToSave1);
+        repository.saveAll(wahltageToSave2);
+        repository.saveAll(wahltageToSave3);
+        List<Wahltag> foundWahltage = repository.findAll();
+        Assertions.assertThat(foundWahltage).containsAll(wahltageToSave1).containsAll(wahltageToSave2).containsAll(wahltageToSave3);
+    }
+
+    @Test
+    void findById() {
+        List<Wahltag> wahltageToSave1 = createWahltagList("X");
+        repository.saveAll(wahltageToSave1);
+        Wahltag wahltag3 = wahltageToSave1.get(2);
+        Optional<Wahltag> foundWahltagById = repository.findById("X_identifikatorWahltag3");
+
+        Assertions.assertThat(foundWahltagById).isPresent();
+        Assertions.assertThat(foundWahltagById.get()).isEqualTo(wahltag3);
+    }
+
+    /**
+     * Tests if searched returned Wahltags are right sorted
+     */
+    @Test
+    @Transactional(propagation = Propagation.REQUIRED, noRollbackFor = Exception.class)
```

Comment:
Die Annotation ist für den Test nicht erforderlich

```suggestion
    @Test
```

Reply:
resolved c1e59f5936ad71788afc2daa780ea211b3def3a6